junit5/src/test/java/org/jboss/weld/junit5/auto/nested/OuterTestClassAndBeanSameInstanceTest.java

```diff
@@ -0,0 +1,81 @@
+package org.jboss.weld.junit5.auto.nested;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Objects;
+
+import javax.enterprise.event.Event;
+import javax.enterprise.event.Observes;
+import javax.enterprise.inject.Any;
+import javax.enterprise.inject.Disposes;
+import javax.enterprise.inject.Produces;
+import javax.inject.Inject;
+
+import org.jboss.weld.junit5.auto.EnableAutoWeld;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests that outer test classes don't get instantiated again for {@code @}{@link Produces Produces},
```

Comment:
Multiple `{@code @}{@link...` occurrences, use one or the other :)

Reply:
That was my own naïve idea how to write annotations into JavaDoc. I changed it the same way now as in cdi-api. Thanks for the hint.